### PR TITLE
changing previous TSG/L1 cards from 13 and 14 TeV to 13p6TeV

### PR DESCRIPTION
**Pull request by msommerh**
Powheg and MadGraph cards. Many of the MG one were taken from inside the previous gridpacks. The center of mass energy was adjusted to 13.6 TeV.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/GF_HH_benchmarks/GF_HH_SM/GF_HH_SM_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+set group_subprocesses Auto 
+set ignore_six_quark_processes False 
+set loop_optimized_output True 
+set complex_mass_scheme False 
+import model BSM_gg_hh
+define lep = e+ e- mu+ mu-
+define nus = ve~ vm~ ve vm
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/GF_HH_benchmarks/GF_HH_SM/GF_HH_SM_proc_card.dat:7]: it seems line 6 and 7 are not used, if so they can be removed.

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/GF_HH_benchmarks/GF_HH_SM/GF_HH_SM_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  False     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  100000 = nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/GF_HH_benchmarks/GF_HH_SM/GF_HH_SM_run_card.dat:32]: for LO, this number will set in the script, but to make it not confusing, please use a smaller value, e.g., 3000

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j_proc_card.dat:36]: the banner can be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1	= run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  500000	= nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j_run_card.dat:28]: for LO, this number will set in the script, but to make it not confusing, please use a smaller value, e.g., 3000

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j_run_card.dat [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+#
+#*********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma) ! Define on/off-shell for "$" and decay  
+#*********************************************************************
+  15.0	= bwcutoff ! (M+/-bwcutoff*Gamma)
+#*********************************************************************
+# Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*********************************************************************
+  False	= cut_decays ! Cut decay products
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j_run_card.dat:108]: it's what you want? if False is used, the cut you defined below will not be applied

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j_run_card.dat [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+  -1	= dynamical_scale_choice ! Choose one of the preselected dynamical choices
+  1.0	= scalefact ! scale factor for event-by-event scales
+#*********************************************************************
+# Type and output format
+#*********************************************************************
+  False	= gridpack !True = setting up the grid pack
+  -1.0	= time_of_flight ! threshold (in mm) below which the invariant livetime is not written (-1 means not written)
+  3.0	= lhe_version ! Change the way clustering information pass to shower.        
+  True	= clusinfo ! include clustering tag in output
+  sum	= event_norm ! average/sum. Normalization of the weight in the LHEF
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma20GeV_MLM_4f_max1j_run_card.dat:68]: why 'sum' is used for normalization, usually average is used

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j_proc_card.dat [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j_proc_card.dat:36]: the banner can be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j_run_card.dat [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1	= run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  500000	= nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j_run_card.dat:28]: for LO, this number will set in the script, but to make it not confusing, please use a smaller value, e.g., 3000

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j_run_card.dat [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+  -1	= dynamical_scale_choice ! Choose one of the preselected dynamical choices
+  1.0	= scalefact ! scale factor for event-by-event scales
+#*********************************************************************
+# Type and output format
+#*********************************************************************
+  False	= gridpack !True = setting up the grid pack
+  -1.0	= time_of_flight ! threshold (in mm) below which the invariant livetime is not written (-1 means not written)
+  3.0	= lhe_version ! Change the way clustering information pass to shower.        
+  True	= clusinfo ! include clustering tag in output
+  sum	= event_norm ! average/sum. Normalization of the weight in the LHEF
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j_run_card.dat:68]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j_run_card.dat [hunk 10]
```diff
@@ -0,0 +1,272 @@
...
+#
+#*********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma) ! Define on/off-shell for "$" and decay  
+#*********************************************************************
+  15.0	= bwcutoff ! (M+/-bwcutoff*Gamma)
+#*********************************************************************
+# Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*********************************************************************
+  False	= cut_decays ! Cut decay products
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma40GeV_MLM_4f_max1j_run_card.dat:108]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j_proc_card.dat [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j_proc_card.dat:36]: the banner can be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j_run_card.dat [hunk 12]
```diff
@@ -0,0 +1,272 @@
...
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1	= run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  500000	= nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j_run_card.dat:28]: for LO, this number will set in the script, but to make it not confusing, please use a smaller value, e.g., 3000

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j_run_card.dat [hunk 13]
```diff
@@ -0,0 +1,272 @@
...
+  -1	= dynamical_scale_choice ! Choose one of the preselected dynamical choices
+  1.0	= scalefact ! scale factor for event-by-event scales
+#*********************************************************************
+# Type and output format
+#*********************************************************************
+  False	= gridpack !True = setting up the grid pack
+  -1.0	= time_of_flight ! threshold (in mm) below which the invariant livetime is not written (-1 means not written)
+  3.0	= lhe_version ! Change the way clustering information pass to shower.        
+  True	= clusinfo ! include clustering tag in output
+  sum	= event_norm ! average/sum. Normalization of the weight in the LHEF
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j_run_card.dat:68]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j_run_card.dat [hunk 14]
```diff
@@ -0,0 +1,272 @@
...
+#
+#*********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma) ! Define on/off-shell for "$" and decay  
+#*********************************************************************
+  15.0	= bwcutoff ! (M+/-bwcutoff*Gamma)
+#*********************************************************************
+# Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*********************************************************************
+  False	= cut_decays ! Cut decay products
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/HAHMHToAA/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j/hToaaTo2b2gamma_ma55GeV_MLM_4f_max1j_run_card.dat:108]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Singlet_Triplet_Higgs_portal_DM/Singlet_Triplet_Higgs_portal_DM_M200dM20ct1m/Singlet_Triplet_Higgs_portal_DM_run_card.dat [hunk 15]
```diff
@@ -0,0 +1,263 @@
...
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15.0  = bwcutoff      ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#**********************************************************
+   False  = cut_decays    ! Cut decay products
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Singlet_Triplet_Higgs_portal_DM/Singlet_Triplet_Higgs_portal_DM_M200dM20ct1m/Singlet_Triplet_Higgs_portal_DM_run_card.dat:86]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Singlet_Triplet_Higgs_portal_DM/Singlet_Triplet_Higgs_portal_DM_M200dM20ct30cm/Singlet_Triplet_Higgs_portal_DM_run_card.dat [hunk 16]
```diff
@@ -0,0 +1,263 @@
...
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15.0  = bwcutoff      ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#**********************************************************
+   False  = cut_decays    ! Cut decay products
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Singlet_Triplet_Higgs_portal_DM/Singlet_Triplet_Higgs_portal_DM_M200dM20ct30cm/Singlet_Triplet_Higgs_portal_DM_run_card.dat:86]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Singlet_Triplet_Higgs_portal_DM/Singlet_Triplet_Higgs_portal_DM_M200dM20ct3cm/Singlet_Triplet_Higgs_portal_DM_run_card.dat [hunk 17]
```diff
@@ -0,0 +1,263 @@
...
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15.0  = bwcutoff      ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#**********************************************************
+   False  = cut_decays    ! Cut decay products
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Singlet_Triplet_Higgs_portal_DM/Singlet_Triplet_Higgs_portal_DM_M200dM20ct3cm/Singlet_Triplet_Higgs_portal_DM_run_card.dat:86]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VBF_HH/VBF_HH_CV_1_C2V_2_C3_1/VBF_HH_CV_1_C2V_2_C3_1_14TeV-madgraph_run_card.dat [hunk 18]
```diff
@@ -0,0 +1,268 @@
...
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  False     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  100000 = nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VBF_HH/VBF_HH_CV_1_C2V_2_C3_1/VBF_HH_CV_1_C2V_2_C3_1_14TeV-madgraph_run_card.dat:32]: for LO, this number will set in the script, but to make it not confusing, please use a smaller value, e.g., 3000

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VectorZPrimeGammaToQQGamma/VectorZPrimeGammaToQQGamma_GpT75_M10/VectorZPrimeGammaToQQGamma_GpT75_M10_proc_card.dat [hunk 19]
```diff
@@ -0,0 +1,41 @@
...
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VectorZPrimeGammaToQQGamma/VectorZPrimeGammaToQQGamma_GpT75_M10/VectorZPrimeGammaToQQGamma_GpT75_M10_proc_card.dat:22]: the banner can be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VectorZPrimeGammaToQQGamma/VectorZPrimeGammaToQQGamma_GpT75_M10/VectorZPrimeGammaToQQGamma_GpT75_M10_run_card.dat [hunk 20]
```diff
@@ -0,0 +1,270 @@
...
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15.0  = bwcutoff ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*************************************************************
+  False = cut_decays ! Cut decay products
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VectorZPrimeGammaToQQGamma/VectorZPrimeGammaToQQGamma_GpT75_M10/VectorZPrimeGammaToQQGamma_GpT75_M10_run_card.dat:99]: if False is used here, the cuts on the jet will not be applied on the two jets from y1 decay, just want to make sure it's you want

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VectorZPrimeToQQ/ZPrimeToQQ_DMsimp_HT600_M100/ZPrimeToQQ_DMsimp_HT600_M100_run_card.dat [hunk 21]
```diff
@@ -0,0 +1,264 @@
...
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15.0	= bwcutoff ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*************************************************************
+  False	= cut_decays ! Cut decay products
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VectorZPrimeToQQ/ZPrimeToQQ_DMsimp_HT600_M100/ZPrimeToQQ_DMsimp_HT600_M100_run_card.dat:98]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VectorZPrimeToQQ/ZPrimeToQQ_DMsimp_HT600_M200/ZPrimeToQQ_DMsimp_HT600_M200_run_card.dat [hunk 22]
```diff
@@ -0,0 +1,264 @@
...
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15.0	= bwcutoff ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*************************************************************
+  False	= cut_decays ! Cut decay products
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VectorZPrimeToQQ/ZPrimeToQQ_DMsimp_HT600_M200/ZPrimeToQQ_DMsimp_HT600_M200_run_card.dat:98]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VectorZPrimeToQQ/ZPrimeToQQ_DMsimp_HT600_M50/ZPrimeToQQ_DMsimp_HT600_M50_run_card.dat [hunk 23]
```diff
@@ -0,0 +1,264 @@
...
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15.0	= bwcutoff ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij/kt_durham cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#*************************************************************
+  False	= cut_decays ! Cut decay products
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/VectorZPrimeToQQ/ZPrimeToQQ_DMsimp_HT600_M50/ZPrimeToQQ_DMsimp_HT600_M50_run_card.dat:98]: same question here

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/tH_b_LH/tH_b_LH_narrow_M650/tH_b_LH_narrow_M650_proc_card.dat [hunk 24]
```diff
@@ -0,0 +1,17 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/tH_b_LH/tH_b_LH_narrow_M650/tH_b_LH_narrow_M650_proc_card.dat:9]: these lines can be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/tH_b_LH/tH_b_LH_narrow_M650/tH_b_LH_narrow_M650_run_card.dat [hunk 25]
```diff
@@ -0,0 +1,273 @@
...
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .false.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  300000 = nevents ! Number of unweighted events requested
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/tH_b_LH/tH_b_LH_narrow_M650/tH_b_LH_narrow_M650_run_card.dat:32]: for LO, this number will set in the script, but to make it not confusing, please use a smaller value, e.g., 3000

**Comment by bbilin** [comment]
Closing the ticket due to timeout, if needed, please reopen.

**Pull request closed without merging**